azurerm/internal/services/appplatform/client/client.go

```diff
@@ -0,0 +1,19 @@
+package client
+
+import (
+	"github.com/Azure/azure-sdk-for-go/services/preview/appplatform/mgmt/2019-05-01-preview/appplatform"
```

Comment:
is there no stable version of this API?

Reply:
currently there is no stable version of appplatform